src/components/Tasks.tsx

```diff
@@ -2,56 +2,159 @@ import { type Task as TaskType } from "./TaskInterface";
 import { useEffect, useState } from "react";
 import Task from "./Task";
 import "./tasks.css";
+import CreateTask from "./CreateTask";
 
 const Tasks = () => {
   const [tasks, setTasks] = useState<TaskType[]>([]);
+  const [openCreateTask, setOpenCreateTask] = useState(false);
+  const categories: string[] = ["work", "life", "school"];
```

Comment:
TypeScript in inferring such types.
 ```suggestion
  const categories = ["work", "life", "school"];
```
Here is the proof:
![image](https://github.com/TheaAleksovDev/ReactConceptsPractice/assets/26555796/d1037087-5bc2-473a-9bbc-a6e581778968)